main.html

```diff
@@ -41,5 +43,159 @@ <h1>I’m Norbert. <br>Glad to see you!</h1>
+                <p class="client">CANOPY</p>
+                <img src="images/mobile_menu_images/Card-works/Counter.jpg" alt="counter">
+                <p class="role">Back End Dev</p>
+                <img src="images/mobile_menu_images/Card-works/Counter.jpg" alt="counter">
+                <p class="year-label">2015</p>
+            </div>
+        </div>
+    <p class="secondary-section">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+    <ul class="language-tags">
+            <li class="html">html</li>
+            <li class="css">css</li>
+            <li class="javaScript">javaScript</li>
+    </ul>
+    <div class="action-button">
+        <p>See Project</p>
+    </div>
+</div>
+</div>
+
+</section>
+
+<section class="about-myself">
+    <div class="top">
+    <h1 class="primary-text">About <br>Myself</h1>
+    <p class="secondary-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="social-media">
+        <div class="supporting-text">
+            <p>LET’S CONNECT</p>
+        </div>
+        <div class="frame">
+            <ul class="social-media-icons">
+                <li><a href="https://github.com/norbiboy1"><img src="images/mobile_menu_images/Github.jpg" alt="Github icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/siko-norbert/"><img src="images/mobile_menu_images/Linkedin-icon.jpg" alt="Linkedin icon"></a></li>
+                <li><a href="#"><img src="images/mobile_menu_images/Vector.jpg" alt="Thumb up icon"></a></li>
+                <li><a href="https://twitter.com/NorbertSiko"><img src="images/mobile_menu_images/Twitter.jpg" alt="Twitter icon"></a></li>
+                <li><a href="https://www.microverse.org"><img src="images/mobile_menu_images/Metaverse.jpg" alt="Microverse icon"></a></li>
+            </ul>
+        </div>
+    </div>
+    <div class="normal-button">
+        <p class="get-my-resume">Get my resume</p>
+    </div>
+</div>
+
+<div class="skill-list">
+    <div class="frame-list">
+        <div class="frame-2">
+            <h1 class="language-list">Languages</h1>
+            <ul class="programming-languages">
+                <li class="JS"><img src="images/mobile_menu_images/JS.jpg" class="JS-icon" alt="JavaScript">JavaScript</li>
+                <li class="HTML"><img src="images/mobile_menu_images/HTML.jpg" class="HTML-icon" alt="HyperText Mark-up Language">HTML</li>
+                <li class="CSS"><img src="images/mobile_menu_images/CSS.jpg" class="CSS-icon" alt="Cascading Style Sheets">CSS</li>
+            </ul>
+        </div>
+    </div>
+    <div class="framework-layout">
+        <h1 class="framework-text">Frameworks</h1>
+    </div>
+    <div class="skill-list-layout">
+        <h1 class="skill-list-text">Skills</h1>
```

Comment:
- [ ] This section of your portfolio is not finished yet. Kindly fix the button width, the position, and alignment of the languages images and text, the font family of the languages, and add the arrow buttons.
![image](https://user-images.githubusercontent.com/61919617/170836179-efafce87-c4f4-4b59-9064-3059aa1cf0a4.png)
![image](https://user-images.githubusercontent.com/61919617/170836334-d23d79ba-3220-4471-898b-db4b4d860cde.png)